default/python/AI/colonization/calculate_stability.py

```diff
@@ -158,15 +159,20 @@ def _evaluate_xenophobia(planet, species) -> float:
     if AIDependencies.Tags.XENOPHOBIC not in species.tags:
         return 0.0
     colonies = get_colonized_planets()
-    relevant_systems = within_n_jumps(planet.systemID, 5) & set(colonies.keys())
-    result = 0.0
     universe = fo.getUniverse()
-    value = get_named_real("XENOPHOBIC_SELF_TARGET_HAPPINESS_COUNT")
+    empire = fo.getEmpire()
+    max_jumps = get_named_real("XENOPHOBIC_MAX_JUMPS")
+    relevant_systems = within_n_jumps(planet.systemID, max_jumps) & set(colonies.keys())
```

Comment:
The `& set(colonies.keys())` most be removed, it restricts the set to systems in which the empire owns at least one planet.